_pages/policies/responding-to-public-disclosure-vulnerabilities.md

```diff
@@ -29,6 +29,8 @@ Scores of `low`, `moderate`, and `high` are subjective.
 
 In the case of an **incident**, the responder has determined for any reason that the vulnerability has _already_ impacted system confidentiality, integrity, or availability. The responder immediately follows the [security incident process](https://handbook.18f.gov/security-incidents/).
 
+Important Note: If the reported issue involves **[cloud.gov] (https://cloud.gov)** and is scored **moderate or higher**, the first responder **must** email **[`cloud-gov-emergency@gsa.gov`] (mailto:cloud-gov-emergency@gsa.gov)** so the [cloud.gov] (https://www.cloud.gov) team can make a determination of impact on their systems and begin **[cloud.gov incident reponse](https://cloud.gov/docs/ops/security-ir-checklist/)** as they see fit.
```

Comment:
Suggest "If the reported issue involves **[cloud.gov](https://cloud.gov)** and is scored **moderate or higher**, the first responder **must** ping `@cloud-gov-team` in TTS Slack and email **[`cloud-gov-emergency@gsa.gov`]"